_data/jobs.yml

```diff
@@ -1,3 +1,6 @@
+---
```

Comment:
```suggestion
```
We shouldn't add this `---` here.

Reply:
It was a suggestion from [`yamllint`](https://github.com/adrienverge/yamllint), I removed it.